### ISSUE TITLE
Crypt hyperplasia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9635]
**Preferred term label:**
Crypt hyperplasia

**Synonyms**

**Definition (free text, please give PubMed ID)**
Crypt hyperplasia

Crypt hyperplasia denotes elongation of the length of the crypts of Lieberkühn, a process that initially precedes villous atrophy. Elongation may be caused by expansion of the lamina propria as a result of the proliferation of stromal cells, an influx of inflammatory cells and tissue remodelling.
PMID: [17021129](https://pubmed.ncbi.nlm.nih.gov/17021129)
**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
613217 
PMID: 18572020  5/5

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
